src/styles/headings.css

```diff
@@ -0,0 +1,17 @@
+h1 {
+	@apply text-2xl font-semibold mb-2;
```

Comment:
is not `text-4xl` instead of  `text-2xl`?

Reply:
There's a media query below this. 2xl is for smaller screens

Reply:
```
@screen md {
	h1 {
		@apply text-4xl;
	}
}
```

Reply:
oww yeah... I have not seen :P

Reply:
😉